packages/@react-spectrum/actiongroup/src/ActionGroup.tsx

```diff
@@ -271,6 +271,7 @@ function ActionGroupItem<T>({item, state, isDisabled, isEmphasized, staticColor,
   isDisabled = isDisabled || state.disabledKeys.has(item.key);
   let isSelected = state.selectionManager.isSelected(item.key);
   let {hoverProps, isHovered} = useHover({isDisabled});
+  let domProps = filterDOMProps(item.props);
```

Comment:
```suggestion
  let domProps = filterDOMProps(item.props);
   delete domProps.id;
```
should we be adding this line as well ? It is similar to the PR that you had shared : https://github.com/adobe/react-spectrum/pull/2655/files

Reply:
That isn't needed here. Tabs uses the id to show/hide panels, so we didn't want users to override that there.